lib/remapify.js

```diff
@@ -85,16 +85,24 @@ eachFile = function eachFile (b, file, pattern) {
     , alias = path.join(pattern.expose || '', relativeFilePath)
     , extStripper = new RegExp('(.*?)(' + b._extensions.join('$|\\') + ')$')
     , splitPath = alias.split(path.sep)
+    , isIndexFile = /index.[js|node]/.test(splitPath[splitPath.length - 1])
```

Comment:
Can we use `b._extensions` instead of hard-coding in the extensions? That will use all the extensions that browserify is enabled for (e.g. `.coffee`, or `.es6`)